cloudify_rest_client/filters.py

```diff
@@ -71,3 +75,34 @@ def list(self, **kwargs):
         response = self.api.get('/filters', params=kwargs)
         return ListResponse([Filter(item) for item in response['items']],
                             response['metadata'])
+
+    def get(self, filter_id):
+        response = self.api.get('/filters/{0}'.format(filter_id))
+        return Filter(response)
+
+    def delete(self, filter_id):
+        self.api.delete('/filters/{0}'.format(filter_id))
+
+    def update(self, filter_id, new_filter_rules=None, new_visibility=None):
+        """Updates the filter's visibility or rules
+
+        :param filter_id: The Id of the filter to update
+        :param new_filter_rules: A new list of filter rules. Filter rules must
+               be one of: <key>=<value>, <key>=[<value1>,<value2>,...],
+               <key>!=<value>, <key>!=[<value1>,<value2>,...], <key> is null,
+               <key> is not null
+        :param new_visibility: The new visibility to update
+        :return: The updated filter
+        """
+        data = {}
+        if not new_filter_rules and not new_visibility:
+            raise RuntimeError('In order to update a filter, you must specify '
```

Comment:
I think this should be the behavior, if no update is done, then throw an error. What do you think?

Reply:
sure, i dont think this is very important